src/+otp/+zlakinetics/mass.m

```diff
@@ -0,0 +1,6 @@
+function M = mass(~, ~, ~, ~, ~, ~, ~, ~)
+
+M = speye(6); 
+M(6,6) = 0;
```

Comment:
Is this faster than just having eye(6)? I know we looked into this for 8 variables.

Reply:
When we tested for transistor amplifier which is similar in size, a sparse mass matrix was slower than dense.